themes/scss/widgets/form-elements.scss

```diff
@@ -11,7 +8,7 @@
   border: none;
   outline: none;
   font-family: 'Open Sans';
-  font-size: $sFontSize-larger;
+  font-size: $sFontSize-large;
```

Comment:
Name of variable `sFontSize-large` should be written in all lowercase letters with hyphens instead of underscores